gears/src/main/resources/nl/lumc/sasc/biopet/pipelines/gears/krakenKrona.ssp

```diff
@@ -15,38 +15,20 @@
 <%@ var args: Map[String, Any] %>
 <%@ var outputDir: File %>
 <%@ var centrifugeTag: Option[String] = None %>
-<%@ var summaryStatsTag: String = "krakenreport" %>
-<%@ var summaryModuleTag: String = "gearskraken" %>
+<%@ var summaryModuleName: String %>
+<%@ var summaryPipelineName: String %>
 <%@ var runId: Int %>
```

Comment:
I think runId is not used anymore here (at least it should not be used anymore ;))